app/models/concerns/orchestration.rb

```diff
@@ -151,6 +151,18 @@ def process(queue_name)
     fail_queue(q)
 
     rollback
+  ensure
+    unless q.nil?
+      logger.info("Processed #{q.completed.count}/#{q.all.count} orchestration tasks") if q.all.count > 0
+      q.all.each do |task|
+        msg = "Task #{task.name} *#{task.status}*"
+        if task.status != "completed"
```

Comment:
Should this also exclude pending tasks? I don't think they shouldn't be considered as errors.